code/drasil-example/swhs/lib/Drasil/SWHS/Requirements.hs

```diff
@@ -160,10 +157,10 @@ reusable = cic "reusable" (foldlSent [
 
 maintainable :: ConceptInstance
 maintainable = cic "maintainable" (foldlSent [
-  S "The traceability between", foldlList Comma List [plural requirement,
-  plural assumption, plural thModel, plural genDefn, plural dataDefn, plural inModel,
-  plural likelyChg, plural unlikelyChg, plural module_], S "is completely recorded in",
-  plural traceyMatrix, S "in the", getAcc srs `S.and_` phrase mg]) "Maintainable" nonFuncReqDom
+  S "If a likely change is made" `S.toThe` S "finished software, it will take", 
+  addPercent (10 :: Integer), S "percent" `S.ofThe` S "original development time,",
+  S "assuming the same development resources are available"
+  ]) "Maintainable" nonFuncReqDom
```

Comment:
This requirement is duplicated a bunch; do we have some place for storing "common" requirements where this could be defined once and reused?